config.json

```diff
@@ -9,11 +9,11 @@
             "type": "amazon-ebs",
             "access_key": "{{user `aws_access_key`}}",
             "secret_key": "{{user `aws_secret_key`}}",
-            "region": "us-east-1",
-            "source_ami": "ami-9711c6fc",
+            "region": "us-west-2",
```

Comment:
Not really. Depends on the use case. If you want to store in default synapse storage may be yes. But still the data transfer speed and cost overhead across regions is very much negligible.